src/Tribe/Aggregator/Record/Queue_Processor.php

```diff
@@ -334,10 +333,12 @@ public static function build_queue( $record, $items = null, $use_legacy = false
 		if ( $record instanceof Tribe__Events__Aggregator__Record__CSV || $use_legacy ) {
 			$class = Tribe__Events__Aggregator__Record__Queue::class;
 		}
-
-		// This is a batch pushing import and should use the Batch Queue to process this import.
+		// If the current Queue is a cron Queue or a Batch Queue.
+		$is_batch_queue  = ( $class === Tribe__Events__Aggregator__Record__Queue::class || $class === Batch_Queue::class );
```

Comment:
* :no_entry_sign: Use Yoda Condition checks, you must.
* :no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: 83769e2f1cd60d9fa6b1acd5b8eb3a11</sub>